pom.xml

```diff
@@ -44,7 +44,7 @@
     <connection>scm:git:git://github.com/jenkinsci/datadog-plugin.git</connection>
     <developerConnection>scm:git:git@github.com:jenkinsci/datadog-plugin.git</developerConnection>
     <url>https://github.com/jenkinsci/datadog-plugin</url>
-    <tag>datadog-3.4.0</tag>
```

Comment:
This one is weird

Reply:
The first release commit changed it:
https://github.com/jenkinsci/datadog-plugin/commit/41bdd039c35c979348ae18d5d61a0403e156acee
The second one changed it back to the original:
https://github.com/jenkinsci/datadog-plugin/commit/06152bf712d8db8dd930be4fa3a64f0fa9508f71

I think something was weird with the way I reverted it which is why it required me to remove it back